src/flags/bolt.hpp

```diff
@@ -26,4 +26,4 @@ DECLARE_string(bolt_cert_file);
 // NOLINTNEXTLINE(cppcoreguidelines-avoid-non-const-global-variables)
 DECLARE_string(bolt_key_file);
 // NOLINTNEXTLINE(cppcoreguidelines-avoid-non-const-global-variables)
-DECLARE_string(bolt_server_name_for_init);
+// DECLARE_string(bolt_server_name_for_init); Moved to run_time_configurable
```

Comment:
Why move any of these? they were already in files related to purpose. Does the runtime configurable mechanism require them to be elsewhere?

Reply:
I didn't want people using them directly, so I hid them in the run time cpp.

Reply:
@Ignition ping